src/utils.php

```diff
@@ -131,3 +132,62 @@ function stripStringOverlap(string $a, string $b): string
     }
     return $b;
 }
+
+/**
+ * Use for sorting an array of URIs by number of segments
+ * in ascending order.
+ *
+ * @param array $uriList
+ * @return void
+ */
+function sortUrisLevelOrder(&$uriList)
+{
+    usort($uriList, function ($a, $b) {
+        return substr_count(Uri\parse($a)['path'], '/') - substr_count(Uri\parse($b)['path'], '/');
+    });
+}
+
+/**
+ * Checks a document against the composer.json to see if it
+ * is a vendored document
+ *
+ * @param PhpDocument    $document
+ * @param \stdClass|null $composerJson
+ * @return bool
+ */
+function isVendored(PhpDocument $document, \stdClass $composerJson = null): bool
+{
+    $path = Uri\parse($document->getUri())['path'];
+    $vendorDir = getVendorDir($composerJson);
+    return strpos($path, "/$vendorDir/") !== false;
+}
+
+/**
+ * Check a given URI against the composer.json to see if it
+ * is a vendored URI
+ *
+ * @param \stdClass|null $composerJson
+ * @param string         $uri
+ * @param array          $matches
+ * @return string
+ */
+function getPackageName(\stdClass $composerJson = null, string $uri): ?string
```

Comment:
can you change this to take `$composerJson` as second parameter, so it's consistent with `isVendored()`?